apps/todo-app/app/globals.css

```
@@ -1,6 +1,11 @@
 @import "tailwindcss";
-@source "../../packages/ui/src/components/ui/**/*.{ts,tsx}";
-@source "../../../node_modules/@lambdacurry/forms/**/*.{js,ts,tsx}";
+
+@source "../**/*.{js,ts,jsx,tsx}";
```

Stylelint (17.2.0)

[error] Unexpected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)